### PR TITLE
added modules for getting a members repos and their links

### PR DESCRIPTION
**Pull request by aribis369**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+    global repos
+    global repolink
+    httpob=urlopen("https://api.github.com/users/"+user_name+"/repos")
+    decob=httpob.read().decode("utf-8")
+    jsonob=json.loads(decob)
+    #print(jsonob)
+    for j in jsonob:
+        if str(j["fork"])=="False":
+            #print(j["full_name"]+"\n")
+            #print("https://www.github.com/"+j["full_name"])
```
Comment (dibyadas) [inline app.py]: @aribis369 Can you remove these comments?

**Review by dibyadas — CHANGES_REQUESTED** [review]
This looks good overall. Just remove those debug comments and add some informative comments for other to follow up.

**Comment by aribis369** [comment]
added comments will you try to render the html file by passing the repo and repolink list to it

File: app.py [hunk 2]
```diff
@@ -2,19 +2,23 @@
...
+
+# module to generate repo and repo links
 def repogen(user_name):
     global repos
     global repolink
+    # getting contents of the api for a user
     httpob=urlopen("https://api.github.com/users/"+user_name+"/repos")
+    # decoding the http object recieved
     decob=httpob.read().decode("utf-8")
+    # covrting to json
```
Comment (dibyadas) [inline app.py]: typo here

**Review by dibyadas — APPROVED** [review]

**Pull request merged**